sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj

```diff
@@ -6,8 +6,7 @@
     <VersionPrefix>4.0.0</VersionPrefix>
     <PackageTags>Microsoft Azure Key Vault Keys</PackageTags>
     <PackageReleaseNotes>
-      <![CDATA[
-      ]]>
+      For release notes and more information please visit https://aka.ms/azure-sdk-preview1-net
```

Comment:
Could you double check that the package will correctly have this in it? Other than that, LGTM!

Reply:
The link currently doesn't work. Is that expected?

Reply:
Yes -- we will update that shortly.

Reply:
VS doesn't show it (they have a bug), but looking at the `nuspec `we have
```
<description>This is the Microsoft Azure Key Vault Secrets client library</description>
<releaseNotes>For release notes and more information please visit https://aka.ms/azure-sdk-preview1-net</releaseNotes>
```

So it looks good